app/classes/report/symbiota.rb

```diff
@@ -36,15 +43,16 @@ def labels
       ]
     end
 
-    def format_row(row)
+    def format_row(row) # rubocop:disable Metrics/AbcSize
       [
         row.name_text_name,
         row.name_author,
         row.name_rank,
         row.genus,
         row.species,
         row.form_or_variety_or_subspecies,
-        *collector_and_number(row),
+        collector(row),
```

Comment:
Changed/added rows so that there are the same number of lines as `labels`;
facilitates side-by-side visual comparison with `labels`.